build/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM golang:1.16-alpine
```

Comment:
Who/what is using this?

Reply:
the k0s build is using that here: 
https://github.com/k0sproject/k0s/blob/790a95678ed9e26c50da98564cc22180f1cb9eaf/Makefile#L63

This is the image that is built in this step:
https://github.com/k0sproject/k0s/blob/790a95678ed9e26c50da98564cc22180f1cb9eaf/Makefile#L60